pilot/lib/pilot/build_manager.rb

```diff
@@ -43,6 +43,10 @@ def upload(options)
 
       UI.success("Successfully uploaded the new binary to App Store Connect")
 
+      unless config[:skip_waiting_for_build_processing]
```

Comment:
Lets remove this block here, this is too specific to give _all_ users of pilot an `important` output here. The change in the options documentation has to be enough.

Reply:
👍